Projects/App/Sources/Register/RegisterGifticonViewModel.swift

```diff
@@ -12,25 +12,74 @@ import RxRelay
 import RxSwift
 
 protocol RegisterGifticonViewModelProtocol {
-    var gifticonImage: UIImage { get set }
+    var gifticonImage: UIImage { get }
     var categories: BehaviorRelay<[String]> { get }
+    var informationValidate: PublishRelay<Bool> { get }
+    var toast: PublishRelay<RegisterGifticonViewModel.Toast> { get }
+    
+    func update(_ type: RegisterGifticonViewModel.Update)
+    func requestRegister()
 }
 
 final class RegisterGifticonViewModel: RegisterGifticonViewModelProtocol {
+    enum Update {
+        case category(Int)
+        case brandName(String?)
+        case productName(String?)
+        case expirationDate(String?)
+        case deadLineMinute(String?)
```

Comment:
```suggestion
        case deadlineMinute(String?)
```
하나의 단어라서 Line이라고 안하셔도 댈거같습니다!

Reply:
수정하겠습니다 😄